include/mbedtls/entropy.h

```diff
@@ -258,6 +258,15 @@ int mbedtls_entropy_update_seed_file( mbedtls_entropy_context *ctx, const char *
  * \return         0 if successful, or 1 if a test failed
  */
 int mbedtls_entropy_self_test( int verbose );
+
+#if !defined(MBEDTLS_TEST_NULL_ENTROPY) && defined(MBEDTLS_ENTROPY_HARDWARE_ALT)
```

Comment:
Checking if MBEDTLS_TEST_NULL_ENTROPY is defined may be a little bit redundant, because these two cannot be defined at the same time: It is made sure in check_config.h. (Yes, I forgot about this when I was making my previous comment about this)